o2a/converter/oozie_converter.py

```diff
@@ -121,6 +128,47 @@ def convert_dependencies(workflow: Workflow) -> None:
         for node in workflow.nodes.values():
             workflow.dependencies.update(node.mapper.required_imports())
 
+    @staticmethod
+    def convert_relations(workflow: Workflow) -> None:
+        """
+        Given a dictionary of task_ids and ParsedActionNodes,
+        returns a set of logical connectives for each task in Airflow.
+
+        :return: Set with strings of task's downstream nodes.
```

Comment:
This comment and :Return: are wrong. The logical connections are stored in workflow.relations and not returned.